zebra-network/src/peer_set/set.rs

```diff
@@ -23,11 +24,17 @@ use tower::{
 use tower_load::Load;
 
 use crate::{
-    protocol::internal::{Request, Response},
+    protocol::{
+        external::InventoryHash,
+        internal::{Request, Response},
+    },
     BoxedStdError,
 };
 
-use super::unready_service::{Error as UnreadyError, UnreadyService};
+use super::{
+    inventory_registry::InventoryRegistry,
```

Comment:
I think it would be slightly cleaner to re-export `InventoryRegistry` from `peer_set` and then import it here as `super::InventoryRegistry` so that module subtrees don't have to know as much about the details of their sibling subtrees.